lib/datadog/grok/src/build.rs

```diff
@@ -0,0 +1,54 @@
+extern crate lalrpop;
+
+use std::fmt;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::{env, fs};
+
+fn main() {
+    lalrpop::Configuration::new()
+        .always_use_colors()
+        .process_current_dir()
+        .unwrap();
+
+    println!("cargo:rerun-if-changed=src/grok.lalrpop");
+
+    read_grok_patterns();
+}
+
+/// Reads grok patterns defined in the `patterns` folder into the static `PATTERNS` variable
+fn read_grok_patterns() {
+    let mut output = String::new();
+
+    fmt::write(
+        &mut output,
+        format_args!("static PATTERNS: &[(&str, &str)] = &[\n"),
+    )
+    .unwrap();
```

Comment:
```suggestion
    let mut output = "static PATTERNS: &[(&str, &str)] = &[\n".to_string();
```

If I'm understanding the usage here correctly, you appear to be starting with a fixed string without requiring interpolation?

Reply:
Yes, nice catch, updated.